### PR TITLE
Cryptocurrency Metrics

### PR DESCRIPTION
**Pull request by MrOmachi**
**In this Project, I have implemented the following:**

- Use React components.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write integration tests with a React testing library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,99 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (youmari) [inline README.md]: - [x] please make sure to add credit to the author of the design as in the requirements 
Screen Shot 2022-10-05 at 16 12 20

File: src/component/Home/CryptoCards.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                      $
+                      {btc.price}
+                    </h2>
+                  </div>
+                </div>
+              );
+            }
+            return null;
+          })}
+        </div>
```
Comment (youmari) [inline src/component/Home/CryptoCards.js:40]: - [x] Nice implementing the requirements but you were asked to follow the design provided so I encourage you to use the same Layout: composition and space between elements. in order to follow the Design 
it should be two cards in a row as in the design 
- screenshot of the requirements 
Screen Shot 2022-10-05 at 16 18 09
- Yours 
Screen Shot 2022-10-05 at 16 15 02
- Design
Screen Shot 2022-10-05 at 16 19 12

File: src/__test__/components.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  it('Testing the NavBar component', () => {
+    const Navbar = render(
+      <BrowserRouter>
+        <NavBar />
+      </BrowserRouter>,
+    );
+
+    expect(Navbar).toMatchSnapshot();
+  });
+});
```
Comment (youmari) [inline src/__test__/components.test.js:46]: - [ ] Great testing your components but you forgot to test the pure function of redux code like action creators so please try to test the pure function in redux code in order to cover those with tests and also to get to the new project 
- Screenshot of the requirements
Screen Shot 2022-10-05 at 16 22 19

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @MrOmachi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (MrOmachi) [reply README.md]: Template owner was acknowledged.

File: src/component/Home/CryptoCards.js (hunk 2 above)
Comment (MrOmachi) [reply src/component/Home/CryptoCards.js:40]: Designed styling now carries two column on the mobile view to suite the provided template as part of the requirement

File: src/__test__/components.test.js (hunk 3 above)
Comment (MrOmachi) [reply src/__test__/components.test.js:46]: Write test for async functions at pending and fulfilled state.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @MrOmachi👋, 

### You did a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- All tests are passed. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**